securedrop_client/gui/widgets.py

```diff
@@ -915,12 +942,19 @@ def __init__(self, source, controller):
 
 
 class TitleLabel(QLabel):
-    """Centered aligned, HTML heading level 3 label."""
+    """The title for a conversation."""
 
     def __init__(self, text):
-        html_text = "<h3>%s</h3>" % (text,)
+        html_text = '<h1>{}</h1>'.format(text)
+        super().__init__(_(html_text))
```

Comment:
Likewise here with the 118n. In this case, we're injecting dynamic text, and this would need to be translated at its source and not here. This isn't something you should fix in this PR because it means going to all the call-sites of this function and making sure all of them translate things correctly. I'm just pointing this out as a way to help you understand how to think about strings in a way that's friendly for i18n. :D

Reply:
Ah, I see. Thanks for pointing this out.